packages/castle/examples/multiple-topics.ts

```diff
@@ -34,14 +35,21 @@ const eachStartEvent = consumeEachMessage<StartEvent>(async ({ message }) => {
 const eachBatchFeedbackEvent = consumeEachBatch<FeedbackEvent>(async ({ batch, producer }) => {
   console.log(`Feedback ${batch.messages.map(msg => `${msg.value.id}:${msg.value.status}`)}`);
   console.log('Sending complete events');
-  sendComplete(producer, batch.messages.map(msg => ({ value: { id: msg.value.id } })));
+  sendComplete(
+    producer,
+    batch.messages.map(msg => ({ value: { id: msg.value.id } })),
+  );
```

Comment:
Wonder why this happened don't see anything changed on those lines, do you have editorconfig setup?

Reply:
My editor is configured to run `prettier` on filesave, if the node project has prettier setup, yeah.

Reply:
It does, but that config is in editorconfig, not prettier. I thought prettier internally used editorconfig, but that might not be the case.

Reply:
Oooo, TIL about editorconfig.
Prettier is [meant to support](https://prettier.io/blog/2017/12/05/1.9.0.html) it yeah
But even using the cli still changes it, also on master.

`yarn prettier --write packages/castle/examples/multiple-topics.ts`